ipv8/attestation/trustchain/block.py

```diff
@@ -376,22 +379,26 @@ def sign(self, key):
         self.signature = self.crypto.create_signature(key, self.pack(signature=False))
 
     @classmethod
-    def create(cls, block_type, transaction, database, public_key, link=None, link_pk=None):
+    def create(cls, block_type, transaction, database, public_key, link=None, additional_info=None, link_pk=None):
         """
         Create an empty next block.
         :param block_type: the type of the block to be constructed
         :param transaction: the transaction to use in this block
         :param database: the database to use as information source
         :param public_key: the public key to use for this block
         :param link: optionally create the block as a linked block to this block
+        :param additional_info: Additional information, which has a higher priority that the
+               transaction when link exists
         :param link_pk: the public key of the counterparty in this transaction
         :return: A newly created block
         """
         blk = database.get_latest(public_key)
         ret = cls()
         if link:
+            import json
```

Comment:
Besides the comment of @qstokkink, only use local imports when it's really necessary 👍